DESCRIPTION

```diff
@@ -0,0 +1,22 @@
+Package: TimeDomainAnalysis
+Title: What the Package Does (Time domain analysis for heart rate variation data.)
+Version: 0.0.1.
+Authors@R: 
+    person(given = "Molly",
+           family = "Rooyakkers",
+           role = c("aut"),
+           email = "molly.rooyakkers@student.uva.nl",)
+
```

Comment:
I would probably link to some paper that you found informative to already give a source for interested readers.